subsys/bluetooth/audio/ascs.c

```diff
@@ -1479,39 +1479,39 @@ int bt_ascs_config_ase(struct bt_conn *conn, struct bt_bap_stream *stream, struc
 	return 0;
 }
 
-static bool codec_config_len_is_valid(struct net_buf_simple *buf)
+static bool ascs_config_check_len(struct net_buf_simple *buf)
 {
-	const struct bt_ascs_config_op *req;
+	const struct bt_ascs_config_op *op;
 	struct net_buf_simple_state state;
 
 	net_buf_simple_save(buf, &state);
 
-	if (buf->len < sizeof(*req)) {
-		LOG_WRN("Malformed ASE Config");
+	if (buf->len < sizeof(*op)) {
+		LOG_WRN("Malformed PDU");
```

Comment:
Perhaps we should log the expected and actual size. Maybe also state "Incorrect length" instead of "Malformed PDU"?